Description: Fix invalid cast from (short) integers to void pointers.

--- a/liveMedia/MPEG1or2Demux.cpp
+++ b/liveMedia/MPEG1or2Demux.cpp
@@ -161,7 +161,7 @@
   // Make sure this stream is not already being read:
   if (out.isCurrentlyAwaitingData) {
     envir() << "MPEG1or2Demux::registerReadInterest(): attempt to read stream id "
-	    << (void*)streamIdTag << " more than once!\n";
+	    << streamIdTag << " more than once!\n";
     envir().internalError();
   }
 
@@ -460,7 +460,7 @@
     skipBytes(pack_stuffing_length);
   } else { // unknown
     fUsingDemux->envir() << "StreamParser::parsePack() saw strange byte "
-			  << (void*)nextByte
+			  << nextByte
 			  << " following pack_start_code\n";
   }
 
--- a/liveMedia/H263plusVideoRTPSink.cpp
+++ b/liveMedia/H263plusVideoRTPSink.cpp
@@ -64,7 +64,7 @@
     }
     if (frameStart[0] != 0 || frameStart[1] != 0) {
       envir() << "H263plusVideoRTPSink::doSpecialFrameHandling(): unexpected non-zero first two bytes: "
-	      << (void*)(frameStart[0]) << "," << (void*)(frameStart[1]) << "\n";
+	      << (frameStart[0]) << "," << (frameStart[1]) << "\n";
     }
     frameStart[0] = specialHeader>>8;
     frameStart[1] = (unsigned char)specialHeader;
